revolv/donor/urls.py

```diff
@@ -1,8 +1,10 @@
 from django.conf.urls import patterns, url
 from django.contrib.auth.decorators import login_required
 from revolv.donor.views import DonorDashboardView
+from revolv.administrator.views import AdministratorAccountingView
 
 urlpatterns = patterns(
     '',
     url(r'^$', login_required(DonorDashboardView.as_view()), name='dashboard'),
+    url(r'^accounting$', AdministratorAccountingView.as_view(), name='accounting'),
```

Comment:
Like we can put it in the main urls.py if needed (which might make sense since it's not specific to donors or admins)